frontend/src/js/external-forms/form-components/DropzoneBetweenElements.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from "@emotion/styled";
+import { DropTargetMonitor, useDrop } from "react-dnd";
+
+import { PossibleDroppableObject } from "../../ui-components/Dropzone";
+
+interface Props<DroppableObject> {
+  onDrop: (item: DroppableObject, monitor: DropTargetMonitor) => void;
+  acceptedDropTypes: string[];
+  lastElement?: boolean;
+  top?: number;
+}
+
+const RootHeightBase = 30;
+const LineHeight = 3;
+const Root = styled("div")`
+  width: 100%;
+  left: 0;
+  right: 0;
+  position: absolute;
+  border-radius: ${({ theme }) => theme.borderRadius};
+`;
+
+const Line = styled("div")<{ show: boolean }>`
+  overflow: hidden;
+  display: block;
+  visibility: ${({ show }) => (show ? "visible" : "hidden")};
+  background-color: ${({ theme }) => theme.col.blueGrayDark};
+  margin: 1px 0;
+  height: ${LineHeight}px;
+  border-radius: 2px;
+`;
+
+const DropzoneBetweenElements = <
+  DroppableObject extends PossibleDroppableObject,
+>({
+  acceptedDropTypes,
+  onDrop,
+  lastElement,
+  top,
+}: Props<DroppableObject>) => {
+  const [{ isOver }, addZoneRef] = useDrop({
+    accept: acceptedDropTypes,
+    drop: onDrop,
+    collect: (monitor) => ({
+      isOver: monitor.isOver(),
+      isDroppable: monitor.canDrop(),
+    }),
+  });
```

Comment:
I changed the Root element to a `Dropzone`.  That works as well. I am not completley convinced on updating the isOver property in the parent component. However, I could not find a better solution. Do you have any ideas on that, or do you think its fine like that?

I chose to only provide top and height for now instead of a classname, since that would cover all cases for now.